### PR TITLE
Require consent rather than 2/3 vote for emeritus

### LINKED ISSUE #713: Criteria to move to Emeritus
PEP recently moved to Emeritus without any vote. Project Progression says there should be a vote. If we do not believe there should be a vote, then we should update Project Progression.

https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#acceptance-criteria-3

Let's discuss. :)

### PR DESCRIPTION
**Pull request by eemeli**
Closes #713

### CODE REVIEW

**Review by sendilkumarn — APPROVED** [review]

File: PROJECT_PROGRESSION.md
Comment (MylesBorins) [inline PROJECT_PROGRESSION.md:216]: Should this be "consensus" rather than consesnt?

**Review by codebytere — APPROVED** [review]

File: PROJECT_PROGRESSION.md
Comment (tobie) [reply PROJECT_PROGRESSION.md:216]: Suggesting the following wording to make your life easier. Feel free to ignore and word differently.

```suggestion
Projects may be granted Emeritus status through CPC consensus and with the approval of the maintainers of the project.
```

File: PROJECT_PROGRESSION.md
Comment (tobie) [inline PROJECT_PROGRESSION.md:217]: Clarifying the concept of "ownership" which isn't really defined anywhere:

```suggestion
In cases where the project maintainers aren't responding to the CPC for six (6) weeks after having been contacted through the appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
I think this changes our governance so we should get board approval.

File: PROJECT_PROGRESSION.md
Comment (eemeli) [reply PROJECT_PROGRESSION.md:217]: I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity. Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Comment (tobie) [reply PROJECT_PROGRESSION.md:217]: > I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity.

Would you prefer:

```suggestion
In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

> Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Yes. I don't see the rationale for making a distinction between those two cases. If anything, Emeritus should be automatic for projects who are no longer maintained.

**Comment by mhdawson** [comment]
@SaraJo and I have this on the agenda for the board meeting tomorrow.

**Comment by mhdawson** [comment]
This was approved by the board in the last meeting. Either with/without the proposed changes from @tobie.

**Comment by mhdawson** [comment]
In the CPC meeting there was consensus to add Tobie's comments and land based on the board approval. We can handle any further discussion/tweaks through a follow on.

**Pull request merged**
